src/main/java/frc/robot/subsystems/BuddyBalanceSubsystem.java

```diff
@@ -75,36 +87,92 @@ public BuddyBalanceSubsystem() {
     tunerNumLeftI = new TunableNumber("Buddy Balance Left Motor I", kDefaultMotorI, leftPIDController::setI);
     tunerNumLeftD = new TunableNumber("Buddy Balance Left Motor D", kDefaultMotorD, leftPIDController::setD);
 
-    refPointBalancedTuner = new TunableNumber("Ref Point Balanced", 15, (a) -> {kBalancedPosition = a;});
-    refPointDeployedTuner = new TunableNumber("Ref Point Deployed", 0, (a) -> {kDeployedPosition = a;});
-    refPointServoTuner = new TunableNumber("Ref Point Servo", 1, (a) -> {kServoDeployedPos = a;});
+    refPointBalancedTuner = new TunableNumber("Ref Point Balanced", kRetrievedAngle, (a) -> {tunedRetrievedAngle = Units.degreesToRadians(a);});
+    refPointDeployedTuner = new TunableNumber("Ref Point Deployed", kDeployedAngle, (a) -> {tunedDeployedAngle = Units.degreesToRadians(a);});
+    refPointServoTuner = new TunableNumber("Ref Point Servo", kServoDeployedPos, (a) -> {tunedServoDeployedPos = Units.degreesToRadians(a);});
+  }
 
-    positionEntry = buddyBalanceTab.add("Buddy Balance Position", 0).getEntry();
+  public double getRightAngle() {
+    return Math.round(AbsoluteEncoder.getPositionRadians(rightEncoder)*100)/100.0;
+  }
+
+  public double getLeftAngle() {
+    return Math.round(AbsoluteEncoder.getPositionRadians(leftEncoder)*100)/100.0;
   }
 
   public boolean getIsDeployed() {
     return isDeployed;
   }
 
   public void deployBuddyBalance() {
-    activateDeploy.set(kServoDeployedPos);
+    activateDeploy.set(kServoDeployedPos); // Change constants to the tuned values while testing (kServoDeployedPos becomes tunedServoDeployedPos)
```

Comment:
There's not much purpose in using a tunable number if you still have to redeploy code when tuning